src/Component/Symfony/Command/StubCommand.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Resource\Symfony\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+abstract class StubCommand extends Command
+{
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        (new SymfonyStyle($input, $output))
+            ->error(
+                \sprintf("To run \"%s\" you need the \"%s\" which is currently not installed.\n\nTry running \"composer require %s\".", static::$defaultName ?? '', 'MakerBundle', 'symfony/maker-bundle --dev'),
+            )
+        ;
+
+        return 0;
```

Comment:
[Exit codes](https://github.com/symfony/symfony/blob/c557647ede0bae07098067bb25c2eaaf49277a5b/src/Symfony/Component/Console/Command/Command.php#L36-L38) consts are available since Symfony 5.4, so it could/should be used over here, but I'm unsure if it's within scope of this PR.


```php
    public const SUCCESS = 0;
    public const FAILURE = 1;
    public const INVALID = 2;
```

```patch
--- return 0;
+++ return Command::SUCCESS;
```